pkg/gcp/actuator/actuator.go

```diff
@@ -238,6 +287,50 @@ func (a *Actuator) sync(ctx context.Context, cr *minterv1.CredentialsRequest) er
 	return nil
 }
 
+// syncWIFSecret makes a time-based token available in a Secret in the namespace of an operator that
+// has supplied the following in the CredentialsRequest:
+// a non-nil spec.CloudTokenString
+// a path to the JWT token: spec.cloudTokenPath
```

Comment:
`cloudTokenPath` is not a hard requirement since we have a default value, correct?